src/server.c

```diff
@@ -566,7 +566,16 @@ dictType objectKeyPointerValueDictType = {
 };
 
 /* Set dictionary type. Keys are SDS strings, values are ot used. */
-dictType setDictType = {
+dictType setDictTypeZ = {
+    dictSdsHash,               /* hash function */
+    NULL,                      /* key dup */
+    NULL,                      /* val dup */
+    dictSdsKeyCompare,         /* key compare */
+    dictSdsDestructor,        /* key destructor */
+    NULL                       /* val destructor */
+};
+
+dictType setDictTypeM = {
```

Comment:
add description of this structure, like for example it is done for  _commandTableDictType_
/* Command table. sds string -> command struct pointer. */